README.md

```diff
@@ -5,7 +5,7 @@ Welcome to the MongoDB Enterprise Kubernetes Operator. The Operator enables easy
 * Ops Manager - our enterprise management, monitoring and backup platform for MongoDB. The Operator can install and manage Ops Manager in Kubernetes for you.
 
 The Operator requires access to one of our database management tools - Ops Manager or Cloud Manager - to deploy MongoDB instances. 
-You may run Ops Manager either inside or outside Kubernetes, or may use Cloud Manager (cloud.mongodb.com) instead.
+You may run Ops Manager either inside or outside Kubernetes, or may use Cloud Manager (Ops Manager-based organization on cloud.mongodb.com, Atlas-based organization will not work) instead.
```

Comment:
```suggestion
You may run Ops Manager either inside or outside Kubernetes, or may use Cloud Manager (Cloud Manager-based organization on cloud.mongodb.com, Atlas-based organization will not work) instead.
```

Reply:
This one also needs to be changed.

Reply:
I don't think that we should have this caveat in the first 10 lines of our readme - can we stick with the reference lower down instead?